hack/generate-manifest-windows.sh

```diff
@@ -111,7 +119,11 @@ pushd $TMP_DIR > /dev/null
 
 BASE=../../default
 if [ "$RUNTIME" == "containerd" ]; then
-    BASE=../../containerd
+    if [ "$OVSRunMode" == "container" ]; then
+        BASE=../../containerd-with-ovs
+    elif [ "$OVSRunMode" == "service" ]; then 
+        BASE=../../containerd
+    fi
```

Comment:
```suggestion
    if $INCLUDE_OVS; then
        BASE=../../containerd-with-ovs
    else
        BASE=../../containerd
    fi
```
This is slightly more readable.